pkg/scheduler/framework/plugins/examples/stateful/stateful.go

```diff
@@ -84,7 +84,7 @@ func (mp *MultipointExample) PreBind(ctx context.Context, state *framework.Cycle
 // New initializes a new plugin and returns it.
 func New(config *runtime.Unknown, _ framework.Handle) (framework.Plugin, error) {
 	if config == nil {
-		klog.Error("MultipointExample configuration cannot be empty")
+		klog.ErrorS(nil, "MultipointExample configuration cannot be empty")
```

Comment:
why do we need to change the function here

Reply:
and add nil as param

Reply:
According to the following rules, Error will be replaced by ErrorS. the first param format  of ErrorS is "type error interface", we can use nil as the first param. the second param is message.
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging

Reply:
I think this is abused, 

> We are not replacing currently used logging library (klog) or the way in which it is used
> We are not preserving full compatibility of logging output
> We are not proposing structuring all logs in Kubernetes

I suggest to change only where is necessary and do bring us some benefit.

Reply:
Sorry, I missed this message.
I respect your suggestion, although my point  is different.
I have fixed it. Thank you very much.
@chendave, any questions, feel free to contact me.